core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,57 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Directly maps to database schema.
+// Do not modify type unless you
+// intend to modify the database schema
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
+	CreatedAt        time.Time
+}
+
+// Directly maps to database schema.
+// Do not modify type unless you
+// intend to modify the database schema
+type dbRawReceipt = evmtypes.Receipt
+
+// Directly maps to database schema.
+// Do not modify type unless you
+// intend to modify the database schema
+//
+// Does not map to a single database table.
+// It's comprised of fields from different tables.
+type dbReceiptPlus struct {
+	ID           uuid.UUID        `db:"id"`
+	Receipt      evmtypes.Receipt `db:"receipt"`
+	FailOnRevert bool             `db:"FailOnRevert"`
+}
+
+func fromDBReceipts(rs []dbReceipt) (receipts []EvmReceipt) {
+	for i := 0; i < len(rs); i++ {
+		receipts = append(receipts, EvmReceipt(rs[i]))
+	}
```

Comment:
This will be marginally more performant:

```suggestion
        receipts = make([]EvmReceipt, len(rs))
	for i := 0; i < len(rs); i++ {
		receipts[i] = EvmReceipt(rs[i])
	}
```